docs/hive/index.md

```diff
@@ -38,12 +37,14 @@ Additional resources on Hive may be purchase two ways. Sponsors (PIs and group l
 Contributors always receive priority access to the resources that they have purchased within one minute with the
 "one-minute guarantee" through the `high` partition.
 
+Finally, Hive has a node available for use in graduate teaching. Email `help@ucdavis.edu` to request information or access to these resources.
+
 ## Hive Administration
 
 Hive hardware and software are administrated by the [HPC@UCD team](https://hpc.ucdavis.edu/people).
 
 ### Current Rates
 
-The rates for Hive can be found [here](https://hpc.ucdavis.edu/rates#hive).
+Rates for Hive can be found [here](https://hpc.ucdavis.edu/rates#hive).
 
-New purchases for Hive can be placed through [Hippo](https://hippo.ucdavis.edu/Hive/product/index).
```

Comment:
The backup documentation also references Hippo. We need to make Hippo purchases work for Hive.